### DISCUSSION TITLE
Runtime Timeout Error in Spawn_npc.py script after adding New map

### DISCUSSION THREAD

**Discussion by HarishKumar051993** [carla-simulator/carla#4317] [General]
Hi All,

I have created a new map using RoadRunner and currently using Carla 9.11 version.

After creating a map, when i try to run the Spawn_npc.py script, Iam getting a Runtime error as below:

WARNING: Version mismatch detected: You are trying to connect to a simulator that might be incompatible with this API
WARNING: Client API version = 0.9.11-5-g4c8f4d5f
WARNING: Simulator API version = 0.9.11-5-g4c8f4d5f-dirty
WARNING: cannot parse georeference: ''. Using default values.

destroying 0 vehicles

destroying 0 walkers

done.
Traceback (most recent call last):
File "spawn_npc.py", line 303, in
main()
File "spawn_npc.py", line 144, in main
spawn_points = world.get_map().get_spawn_points()
RuntimeError: time-out of 10000ms while waiting for the simulator, make sure the simulator is ready and connected to 127.0.0.1:2000

The above error i am facing only in LINUX (Not in Windows). And all other scripts as Automaticcontrol.py and manual control.py is working fine in Linux machine.
This Error iam getting only to the new map which i created. In other Town Maps, the Spawn_npc.py script is working.

Can anyone help why iam facing this error.

**Comment by corkyw10** [comment]
Hello, could you increase the timeout to see if the error persists after that?

**Reply by HarishKumar051993** [reply]
@corkyw10 Thanks for the response.

Yeah I tried increasing the Timeout as well. Still observes same issue.
